data/coalition/coalition missions.txt

```diff
@@ -2335,3 +2335,506 @@ mission "Coalition: Submarines 4"
+			`	He tries to push his way to her, but the guards stop him before he can make much progress. One of the Heliarch representatives gets closer, saying, "A restricted area, this is. Invading government property, you are, so suggest that you leave immediately, I do."`
+			`	"Restricted? What about the outside? Is it not also restricted? Much more dangerous? Fine with sending my sister there, you seem," Repilor counters, and the Heliarch looks away after a moment. "If bothering the consuls so much, she was, just fired, she should've been!" He then looks at you, and continues, "And the human? Your transport out of here, I imagine? Or maybe the one who put these ideas in your head, they are?"`
+			choice
+				`	"I've just been hired to transport her."`
+					goto hired
+				`	"What's so wrong about her wanting to explore outside the Coalition?"`
+					goto wrong
+				`	"Wait, what did you say about the outside being dangerous?"`
+			`	The Saryd looks at you, puzzled. Then, as if realizing something, turns to Sedlitaris, "Told the human, you haven't?"`
+			`	"That's enough Repilor. Trespassing on a restricted area, you are. Escort him to his ship," she tells the guards, and after exchanging looks briefly, and after some struggle on Repilor's part, the Arachi manage to push him back inside the Sojourner. "Ignore that you should, Captain. Always... difficult, my brother was. Too stubborn to realize when to stop, and easy to impress with folk tales of the outside."`
+			choice
+				`	"If you say so. Should I wait in my ship until you all are done here?"`
+					goto accept
+				`	"Folk tales or not, I'm not really willing to find out after that. He didn't seem to be lying."`
+			`	"What?" she exclaims. "Please Captain, fall for my brother's little rant, you cannot. Finally permission to leave, I have!"`
+			choice
+				`	"Alright, alright. But I hope I don't regret this."`
+				`	"I'm sorry. Whatever it is he was afraid of, I'm not willing to risk it."`
+					goto decline
+			`	"You won't." She says, heading to you and holding your hands with hers, as if in thanks. "Promise you I do, fine we will be, Captain."`
+				goto accept
+			label decline
+			`	You hear her stomping her hoofs on the ground as you head back to your ship, and see her and the Heliarch council representatives arguing with one another as you close your hatch. When you're back on the spaceport landing area, a Sojourner lands next to your ship, and Repilor, the Saryd from earlier, comes out.`
+			`	"Forgive me for being rude earlier, you must." He says as he greets you. "Thought I had, that talked my sister into this you had. That filled her head with more grandiose tales of the outside, you did." He sighs. "Thank you, Captain. Hate you now, my sister must, but the right thing, you did. Better for the both of you, this is." He nods lightly in a bow, and heads back to his ship.`
```

Comment:
```suggestion
			`	"Forgive me for being rude earlier, you must." He says as he greets you. "Thought I had, that talked my sister into this you did. That filled her head with more grandiose tales of the outside, you must have." He sighs. "Thank you, Captain. Hate you now, my sister must, but the right thing, you did.
			`	"That no inspector has ever returned from their expedition, the truth is, so afraid you were leading my sister to her death, I was. Better for the both of you, this is." He nods lightly in a bow, and heads back to his ship.`
```
Probably should have some payoff for the `"Told the human, you haven't?"` line here. You could even add an option to ask Sedlitaris about what he was talking about later and have her just sort of brush it off, but not necessary in that case since the player will learn at the end anyway. Feel free to edit the suggestion as you see fit though.

Reply:
I'm not sure on giving a choice to call Sedlitaris out on that line from Repilor, since I'd need some good deal more label branching to make sure it only showed up if you picked the choices that make Repilor say that.
Added a bit at the end of this line to try for a payoff, let me know if you meant something other than what I did.